assets/javascripts/audit_log.js

```diff
@@ -309,6 +309,35 @@ function loadProductLogTable(dataTableUrl, rescheduleUrlTemplate, showActions) {
         }
       },
       {targets: 2, orderable: false},
+      {
+        targets: 6,
+        render: function (data, type, row) {
+          const rowData = dataForLink(this);
+          const settings = rowData.settings;
+          const version = settings['VERSION'];
+          const build = settings['BUILD'];
+          const distri = settings['DISTRI'];
```

Comment:
```suggestion
          const version = settings.VERSION;
          const build = settings.BUILD;
          const distri = settings.DISTRI;
```
The same counts the the other definition.